checkov/terraform/checks/resource/azure/ACRUseSignedImages.py

```diff
@@ -7,17 +7,20 @@
 class ACRUseSignedImages(BaseResourceValueCheck):
 
     def __init__(self):
-        # IN ARM - Set properties.trustPolicy.status to enabled, set
-        # properties.trustPolicy.type to Notary
+        # IN ARM - Set properties.policies.trustPolicy.status to enabled, set
+        # properties.policies.trustPolicy.type to Notary
         # This is the default behaviour by the tf provider when the trust policy is enabled
         name = "Ensures that ACR uses signed/trusted images"
         id = "CKV_AZURE_164"
         supported_resources = ("azurerm_container_registry",)
         categories = (CheckCategories.GENERAL_SECURITY,)
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
-    def get_inspected_key(self):
-        return "trust_policy/[0]/enabled"
```

Comment:
Thanks @paddymorgan84! Same comment here. This is very helpful and greatly appreciated, but please keep it backwards compatible by checking for either configuration.